boards/shields/max31790/max31790.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 SILA Embedded Solutions GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_i2c {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	max31790: max31790@20 {
```

Comment:
[to avoid name conflicts with devices that may be defined at board level, it is advised, specifically for shields devicetree descriptions, to provide a device nodelabel is the form `<device>_<shield>`](https://docs.zephyrproject.org/latest/hardware/porting/shields.html#shield-porting-and-configuration)

Reply:
done